src/core/tests/messages.test.js

```diff
@@ -0,0 +1,23 @@
+import { expect, it } from "vitest";
+import { messages } from "../messages";
+
+it("should have unique messages regardless of the emojis", () => {
+  const emojiRegex =
```

Comment:
I think it'll be better to change this regex to something we should accept, instead of something we should not. For example, instead of checking that this it emoji, we could check that this text contains letters, or punctuation symbols and remove everything that's not the letter or punctuation symbol.

That way the regular expression will be probably more readable

Reply:
Apologies on the conflict between branches. I was not careful when I was switching between them. I will fix this right after https://github.com/pavelkeyzik/pr-approve-generator/pull/5 gets merged. 

>  regex to something we should accept, instead of something we should not

This makes a lot more sense! This way I will ensure the regular expression more clear